### ISSUE TITLE
Provide installation instructions for using the latest master-branch version

### ISSUE THREAD

**Issue by JanSurft** [gregorrr/anki-auto-markdown#8]
It would be great to add instructions on how to use the master branch of this plugin instead of the one offered at ankiweb, this way we can be sure that people know how to use the master branch if they want to see if a particular bug/error persists in the latest master version.

**Comment by gregorrr** [comment]
Good idea. It would be great to have some testers! I'll get to this soon

**Comment by ueoo** [comment]
Any updates?
